recipes/eko/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "eko" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/N3PDF/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f7f6d7faed207bbbdbbbf003348e1afdcf823196ecea0156730f0097a3686ef3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py<38]
+
+requirements:
+  host:
+    - python
+    - poetry
+    - pip
+  run:
+    - python
+    - scipy
+    - lz4
+    - numba
+    - pyyaml
+    - numpy
+
+test:
+  imports:
+    - eko
```

```suggestion
    - eko
    - ekobox
    - ekomark
```
Generated using grayskull.